pages/examples/_.vue

```diff
@@ -142,7 +144,7 @@ export default {
       />
       <div v-if="mediaObjectIsEmpty" class="main-content-article-wrapper">
         <article class="main-content-article hide-scroll">
-          <h1 class="main-content-title">{{ exampleItem.title }}</h1>
+          <MainContentHeader :title="exampleItem.title" />
```

Comment:
Using the component on this page template actually updated the page title size of the sub pages under 'media' to match the rest of the pages. Which I think is what we'd want anyway.